model_compiler/python/nnef_to_nnir.py

```diff
@@ -236,7 +236,7 @@ def nnef_op_to_ir_node(nnef_graph, nnef_operation):
     
 def nnef_graph_to_ir_graph(nnef_graph):
 
-    graph = IrGraph()
+    graph = IrGraph(False)
```

Comment:
Codacy found an issue: ['IrGraph' may be undefined, or defined from star imports: nnir (F999)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5754497)